index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="style.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Odin Recipes</h1>
+    <ul>
+        <li><a href="white.html">white-rice</a> </li>
+        <li><a href="rosemary.html">rosemary</a></li>
+        <li><a href="sufganiyot.html">sufganiyot</a></li>
+    </ul>
+    <h1>
+        Rosemary Potatoes with Caramelized Onions
+        <img src="../images/rosemary.jpg">
+    </h1>
+    <h2>How to prepare</h2>
+    <p>This is one of meals that that needs time to prepare as it contains more steps to be followed.</p>
+<h3>Ingredients</h3>
+<ul>
+    <il>2 pounds small red potatoes quartered
+    </il>
+<il>2 garlic cloves minced</il>
+<il>1 tablespoon olive oi</il>
+<il>2 teaspoons minced fresh rosemary or 1/2</il>
+<il>teaspoon dried rosemary crushed</il>
+<il>1/2 teaspoon minced fresh thyme or 1/8</il>
+<il>teaspoon dried thyme</il>
+<il>1/4 teaspoon salt</il>
+<il>1/4 teaspoon pepper</il>
+<il>2 large sweet onions chopped</il>
+<il>2 tablespoons olive oil</il>
+<il>1 tablespoon sugar</il>
+<il>2 teaspoons balsamic vinegar</il>
```

Comment:
The item tag is <li> not **<il>** so it would be better to implement those changes.